api/groups.go

```diff
@@ -5,16 +5,56 @@ package api
 
 import (
 	"context"
+
+	abstractions "github.com/microsoft/kiota-abstractions-go"
+	"github.com/microsoftgraph/msgraph-sdk-go/groups"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
 )
 
 type GroupsClient interface {
 	AddGroupMember(ctx context.Context, groupObjectID string, memberObjectID string) error
 	RemoveGroupMember(ctx context.Context, groupObjectID, memberObjectID string) error
-	GetGroup(ctx context.Context, objectID string) (result Group, err error)
-	ListGroups(ctx context.Context, filter string) (result []Group, err error)
+	GetGroup(ctx context.Context, objectID string) (result models.Groupable, err error)
+	ListGroups(ctx context.Context, filter string) (result []models.Groupable, err error)
 }
 
 type Group struct {
 	ID          string
 	DisplayName string
 }
+
+func (c *AppClient) AddGroupMember(ctx context.Context, groupObjectID string, memberObjectID string) error {
```

Comment:
Should we be using `memberObjectID` in the request somewhere?

Reply:
Yup! Forgot to encode it as part of the `oData`. Fixed now